### PR TITLE
Distributed load balancer cache

### PR DESCRIPTION
**Pull request by jandadav**
Signed-off-by: jandadav <janda.david@gmail.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/extend/extend-apiml/custom-metadata.md [hunk 1]
```diff
@@ -54,21 +54,28 @@
...
+
+    The header name is `X-InstanceId` and sample value is `discoverable-client:discoverableclient:10012`. This is identical to `instanceId` property in Discovery service's registration.
     
-    * **`authentication`**
+    In combination with enabling [Routed instance header](../../user-guide/api-mediation/api-gateway-configuration.md#routed-instance-header), the client can achieve sticky session functionality. The benefit of this approach is that there is no session on Gateway and the client is in full control over whether he wants to go to a specific instance or not. 
+    
+    1) The client calls API Gateway and gets routed to a service
```
Comment (janan07) [inline docs/extend/extend-apiml/custom-metadata.md:68]: Is this a procedure? Could this be a bulleted list?

File: docs/extend/extend-apiml/custom-metadata.md [hunk 2]
```diff
@@ -54,21 +54,28 @@
...
+    2) The client reads the `X-InstanceId` header value from the response to understand the service was routed to
+    3) On all subsequent requests, client provides the `X-InstanceId` header with previously read value to get routed to the same instance of service
     
-    Use this value if the user is authenticated. The instance information is cached and 
-    used afterwards to forward the client request to the specified target server
+    * **`authentication`**
+
+   This applies the Authentication load balancing schema. This is sticky session functionality based on user's ID. The user's ID is understood from Zowe SSO token on client's request. Requests without the token will be routed in round robin fashion. The user is first routed in round robin fashion and then the routed instance Id is cached. The instance information is used for subsequent requests to route the client to the cached target service instance. This session's default expiration time is 8 hours. After the session expires, the process initiates again.
```
Comment (janan07) [inline docs/extend/extend-apiml/custom-metadata.md:74]: Do we need to describe what is a "sticky session"?

Comment (jandadav) [reply docs/extend/extend-apiml/custom-metadata.md]: I think it's universally known term, but I can describe it shortly.

**Review by janan07 — APPROVED** [review]
Reviewed and edited by Tech Info

**Pull request merged**